examples/housing_market_research/investor_analytics/portfolio_analysis.ipynb

```diff
@@ -0,0 +1,864 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<center>\n",
+    "<h1>Welcome to the Lab 🥼🧪</h1>\n",
+    "</center>\n",
+    "\n",
+    "### In this notebook, we will analyze investor ownership concentration in the Milwaukee metropolitan area.\n",
+    "\n",
+    "In this notebook, we will be examining investors in the Milwaukee metro housing market, one of the metropolitan markets where large institutional investors slashed prices of their inventory [last year](https://www.parcllabs.com/articles/where-are-large-operators-cutting-home-prices). We will download information to understand the concentration of single-family homes for rental (SFRs) based on portfolio size. Specifically, we will look at the share of total housing stock by portfolio size to understand local trends.\n",
+    "\n",
+    "#### Need help getting started?\n",
+    "\n",
+    "As a reminder, you can get your Parcl Labs API key [here](https://dashboard.parcllabs.com/signup) to follow along.\n",
+    "\n",
+    "To run this immediately, you can use Google Colab. Remember, you must set your `PARCL_LABS_API_KEY`.\n",
+    "\n",
+    "Run in Colab --> [![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ParclLabs/parcllabs-cookbook/blob/main/examples/housing_market_research/investor_analytics/portfolio_analysis.ipynb)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# if needed, install and/or upgrade to the latest verison of the Parcl Labs Python library\n",
+    "%pip install --upgrade parcllabs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After installing the required libraries, we need to load them and instantiate the Parcl Labs client. The client is a Python library designed to facilitate and optimize the user experience with the Parcl Labs API. It takes care of searching, retrieving, and formatting the data for us. To use the client, you need to have an `API_KEY` available in your [dashboard](https://dashboard.parcllabs.com/). While you can simply enter your `API_KEY` directly, it is recommended that you save it as an environment variable to make it more secure. If you are using Colab, you can follow these [steps](https://medium.com/@parthdasawant/how-to-use-secrets-in-google-colab-450c38e3ec75).\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Environment setup\n",
+    "import os\n",
+    "from io import BytesIO\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import matplotlib.dates as mdates\n",
+    "import plotly\n",
+    "from PIL import Image\n",
```

Comment:
Lets use `from parcllabs.beta.charting.utils import create_labs_logo_dict` instead of introducing a new dependency and remove the PIL imports and usage. 

Can be added to a plotly chart with: 

`fig.add_layout_image(create_labs_logo_dict())`

See [here](https://github.com/ParclLabs/parcllabs-cookbook/blob/main/examples/experimental/supply_and_demand/us_pricefeed_vs_ticker_symbol.ipynb) for an example.

Reply:
Remove PIL dependency.

Reply:
removed branding for this case as discussed and eliminated `PIL` dependency.